Dockerfile

```diff
@@ -96,15 +96,18 @@ ENV LOG_LEVEL="info"\
     DEDUPLICATION_TIME_LIMIT="3600" \
     CENTRAL_BRANCH=master \
     # to enable localstack set this to 1
-    AWS_CBOR_DISABLE=0
+    AWS_CBOR_DISABLE=0 \
```

Comment:
Are these default variables? I'm wondering if we should define those as variable instead and have the values defined in the helm chart value files.

Reply:
AWS_CBOR_DISABLE is used for localstack which is not currently being used but for all other variables with default values as in dockerfile can be overwritten in the value file for helm charts